test/development/acceptance-app/ReactRefreshLogBox-builtins.test.ts

```diff
@@ -58,6 +58,8 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox app %s', () => {
             |             ^^^^^^^^^^^^^^
           2 | module.exports = dns
 
+        Import map: No import map entry
```

Comment:
I don't think we should show this to users

Reply:
I agree, but it just appears somehow?